.devcontainer/default/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "SourceKit LSP",
+  "dockerFile": "Dockerfile",
+
+  // Allow the processes in the container to attach a debugger
+  "capAdd": [ "SYS_PTRACE" ],
+  "securityOpt": [ "seccomp=unconfined" ],
+
+  "mounts": [
+    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance?WT.mc_id=javascript-14373-yolasors#_use-a-targeted-named-volume)
+    "source=${localWorkspaceFolderBasename}-build,target=${containerWorkspaceFolder}/build,type=volume"
+  ],
+  "remoteEnv": {
+    // `/usr/bin` must be first: https://github.com/apple/sourcekit-lsp/blob/main/Documentation/Development.md#linux
+    "PATH": "/usr/bin:${containerWorkspaceFolder}/.build/bin:${containerEnv:PATH}" },
```

Comment:
This also isn’t necessary anymore. SourceKit-LSP now prefers a toolchain if it is a strict superset of another toolchain: https://github.com/apple/sourcekit-lsp/pull/1034